src/utils/html.js

```diff
@@ -75,8 +75,12 @@ function checkFileTypes (types, url) {
 }
 
 u.isAudioURL = url => checkFileTypes(['.ogg', '.mp3', '.m4a'], url);
-u.isImageURL = url => checkFileTypes(['.jpg', '.jpeg', '.png', '.gif', '.bmp', '.tiff', '.svg'], url);
 u.isVideoURL = url => checkFileTypes(['.mp4', '.webm'], url);
+u.isImageURL = url => checkFileTypes(['.jpg', '.jpeg', '.png', '.gif', '.bmp', '.tiff', '.svg'], url);
+u.isWhitelistedImageURL = url => {
+    const regex = _converse.api.settings.get('image_urls_regex');
+    return regex ? regex.test(url) : false;
+}
```

Comment:
I don't see any place where `isImageURL` is called without also having to call `isWhitelistedImageURL`, so it seems unnecessary to me to have two methods.
Instead, `isImageURL` can also do the regex test.

If it ever were necessary to not check for the regex, we could disable that by passing in a boolean to `isImageURL`.

Reply:
Fixed.